src/test/java/cat/udl/eps/softarch/myroutes/steps/RouteVersionStepDefs.java

```diff
@@ -0,0 +1,68 @@
+package cat.udl.eps.softarch.myroutes.steps;
+
+import cat.udl.eps.softarch.myroutes.domain.Coordinate;
+import cat.udl.eps.softarch.myroutes.domain.Route;
+import cat.udl.eps.softarch.myroutes.domain.RouteVersion;
+import cat.udl.eps.softarch.myroutes.domain.User;
+import cat.udl.eps.softarch.myroutes.repository.CoordinateRepository;
+import cat.udl.eps.softarch.myroutes.repository.RouteRepository;
+import cat.udl.eps.softarch.myroutes.repository.RouteVersionRepository;
+import cat.udl.eps.softarch.myroutes.repository.UserRepository;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.When;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+
+import java.nio.charset.StandardCharsets;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+
+public class RouteVersionStepDefs {
+    @Autowired
+    private StepDefs stepDefs;
+
+    @Autowired
+    private RouteVersionRepository routeVersionRepository;
+
+    @Autowired
+    private RouteRepository routeRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @When("I create a new correct RouteVersion")
+    public void iCreateANewCorrectRouteVersion() throws Throwable {
+
+        Route route = routeRepository.findByTitle("testRoute").get(0);
```

Comment:
To make it easier to reuse this step, it would be more convenient to make the route title a step parameter instead of hardcoding it